ui/round/src/ctrl.ts

```diff
@@ -702,7 +702,6 @@ export default class RoundController implements MoveRootCtrl {
 
   goBerserk = () => {
     this.socket.berserk();
-    lichess.sound.play('berserk');
```

Comment:
why remove the sound?

Reply:
Sound is already there in the function that this.socket.berserk() calls which is setBersek() just below this function. I am not sure why the sound was at 2 places. It works correctly after removing this code. You can also hear the sound in the video that I posted.

Reply:
the sound is here so that it doesn't await the websocket network round trip, I think

Reply:
Ok, that means I need to add additional checks when calling goBerserk to know whether the player has already berserked or not. Or the player would be able hear the sound even if they have already berserked.

Reply:
@ornicar Does that mean if the connection is slow the user may hear the berserk sound twice?